generators/src/main/java/kafka/pageclickgenerator/Main.java

```diff
@@ -9,6 +9,7 @@ public static void main(String[] args) throws Exception {
         String bootstrapServer = KafkaUtils.BOOTSTRAP_SERVERS;
         String topic = KafkaUtils.TOPIC;
         String urlsPath = "generators/src/main/resources/urls.csv";
+        String cookieEmailMappingsPath = "generators/src/main/resources/userCookieEmailMappings.csv";
         if (args.length >= 2) {
             bootstrapServer = args[0];
             topic = args[1];
```

Comment:
Should we add possibility to get cookieEmailMappingsPath from program arguments like we do with bootstrapServer, topic and urlsPath? Or will we use this class only from IDE?
P.S. This comment refers to lines 12-19, but Github doesn't allow me to add so many lines that aren't in commit.

Reply:
Slava said that we'll refactor it later